README.md

```diff
@@ -35,195 +26,101 @@ https://simplereport.gov/
+If you wish to run outside of docker-compose (bare metal), you'll need to install all the dependencies locally.
+See more [info here](https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker).
 
-```
-simple-report:
-  demo-users:
-    default-user:
-      authorization:
-        granted-roles: ADMIN
-```
+## Roles
 
-current role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
+The available user role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
 
-`ADMIN` - an organization admin with full access to their organization
-`USER` - a site user the has access to everything in their organization but the gear icon
-`ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
-`ALL_FACILITIES` - a site user that can access all facilities in their organization
-`NO_ACCESS` - a member of an organization who has no permissions without possessing other roles
+- `ADMIN` - an organization admin with full access to their organization
+- `USER` - a site user the has access to everything in their organization but the gear icon
+- `ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
+- `ALL_FACILITIES` - a site user that can access all facilities in their organization
+- `NO_ACCESS` - a member of an organization who has no permissions without possessing other roles. Every member of an org has this role, so it is used to list all users in an organization
 
-#### Site roles
+These roles are controlled via Okta groups.
 
-You can make the default user a site admin by adding the following to `application-local.yaml`:
+## Support admin
 
-```
-simple-report:
-  demo-users:
-    site-admin-emails:
-      - bob@example.com
-```
+You can make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
 
-Site admins can access the `/admin` paths and site admin APIs
+Support admins can access the `/admin` paths and support admin APIs.
 
-### Restart & Clean
+## Database cleaning
 
 When there are DB schema changes the backend may throw an error and fail to start.
 
 Restarting the docker way:
 
-1. run `cd backend`
-1. Bring down the service by running `docker-compose down`
-1. Wipe the db by running `docker system prune && docker images prune && docker volume prune`
-1. Restart the service `docker-compose up --build`
+1. `docker system prune --volumes`
 
 Restarting the SQL way:
 
-1. run `db-setup/nuke-db.sh`
-2. restart the spring app `gradle bootRun --args='--spring.profiles.active=dev'`
+1. `yarn db` to open up a psql prompt
+1. `DROP DATABASE simple_report; CREATE DATABASE simple_report`
```

Comment:
This is going to break things with Liquibase and the `simple_report_no_phi` user, since dropping the DB doesn't drop that user and Liquibase is going to try to recreate it once the `databasechangelog` is missing. The ideal thing to do would be to truncate all the tables instead.

Reply:
I'm going to remove this since it's simple enough to just do a `docker-compose down --volumes`

Reply:
Perfect!